app/assets/javascripts/tylium/modules/resource_sidebar.js

```diff
@@ -0,0 +1,24 @@
+document.addEventListener('turbolinks:load', function() {
+
+  if ($('[data-behavior~=resource-sidebar]').length) {
+    const $resourceSidebar = $('[data-behavior~=resource-sidebar]'),
+          $resourceSidebarToggle = $('[data-behavior~=resource-sidebar-toggle]'),
+          localStorageKey = 'project.ce.resource-sidebar-collapsed',
```

Comment:
Let's have this key in an HTML data attribute, similar to the datatable storage keys for consistency.